config/index.js

```diff
@@ -19,7 +21,11 @@ nconf.defaults({
   SITE_URL: process.env.SITE_URL || 'http://0.0.0.0:4000',
   WP_URL: process.env.WP_URL || 'http://127.0.0.1:6000',
   WP_TOKEN: process.env.WP_TOKEN || '',
-  THEME_DIR: process.env.THEME_DIR || 'themes'
+  THEME_DIR: process.env.THEME_DIR || 'themes',
+  NODE_MODULES_PATH: process.env.NODE_MODULES_PATH || 'node_modules',
+  CKAN_FE_PLUGINS: process.env.CKAN_FE_PLUGINS || '',
+  CKAN_PLUGIN_DIRECTORY: process.env.CKAN_PLUGIN_DIRECTORY || 'plugins',
+  CKAN_THEME_ROUTES: process.env.CKAN_THEME_ROUTES || '',
```

Comment:
We need to have single env var for this and I think `THEME` is the better one since it is more generic @starsinmypockets